app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,115 @@
+<form class="ui form" {{action 'submit' on='submit' }}>
+  <div class="ui segments">
+    <div class="ui aligned secondary segment">
+      <h3 class="weight-400">{{t 'Registration Information'}}</h3>
+    </div>
+    <div class="ui padded segment">
+      <div class="ui yellow inverted segment">
+        <div class="ui tablet stackable grid">
+          <div class="three wide column" style="font-size:24px;margin:10px 0 0 0">
```

Comment:
@harshitagupta30 Can you please move all your inline css to a separate css file?